datajunction-server/datajunction_server/database/availabilitystate.py

```diff
@@ -31,6 +31,7 @@ class AvailabilityState(Base):  # pylint: disable=too-few-public-methods
     table: Mapped[str]
     valid_through_ts: Mapped[int] = mapped_column(sa.BigInteger())
     url: Mapped[Optional[str]]
```

Comment:
I wonder if we can actually remove `url` now, since that was just a single link, but with this change we'll have the option for multiple, and all named as well. Although there will be a database migration involved if we do remove.

Reply:
I was thinking about it. And I noticed that the top level `url` attribute is typically used as a link to a canonical catalog for a particular dataset, which could make sense to be separate. But then it should be called `catalog_url` so maybe I should just rename it, if you agree with this idea?

Reply:
Actually `dataset_url` sounds more intuitive than `catalog_url`. What do you think?

Reply:
I'll skip this for now.